tests/link/numba/utils.py

```diff
@@ -211,11 +211,15 @@ def compare_rv_with_numba_output(fgraph, inputs, rng, size):
 
     inputs = tuple([i.tolist() for i in inputs])
 
+    @numba.vectorize
+    def numba_rng_fn(a):
+        return numpy_fn(*inputs)
```

Comment:
I tried doing a `numba.vectorize` but it still doesn't work with array inputs. There's probably something else I'm missing here.

Also a bit hackish but it seems functions compiled with `numba.vectorize` (without the decorator defining types, probably), they don't accept functions without any arguments. So I passed a single dummy variable(that `a`). Is this an issue with Numba or is it supposed to work this way.

Reply:
The body of the function is supposed to represent the _scalar_ operation that's broadcast across arrays (i.e. "vectorized").

Reply:
> Varargs (i.e. *inputs) might be an issue for numba.vectorize, though.

Yeah, that's a problem. We can just use the `make_numba_random_fn` in this case, If our goal at the end is just comparing the inputs with and without the Numba function being in the graph.